internal/assertions/file_content_assertion.go

```diff
@@ -35,7 +35,7 @@ func (t FileContentAssertion) Run(screenState [][]string, startRowIndex int) (pr
 		return processedRowCount, &AssertionError{
 			StartRowIndex: startRowIndex,
 			ErrorRowIndex: startRowIndex,
-			Message:       fmt.Sprintf("Expected file content (%q) but received %q", t.ExpectedContent, fileContent),
+			Message:       fmt.Sprintf("Expected %s contains (%q) but received %q", t.FilePath, t.ExpectedContent, fileContent),
```

Comment:
```suggestion
			Message:       fmt.Sprintf("Expected %s to contain %q but received %q", t.FilePath, t.ExpectedContent, fileContent),
```

Reply:
(putting the content in brackets doesn't make sense since it's part of the sentence)